### ISSUE TITLE
tensorflow issue

### ISSUE THREAD

**Issue by timombre** [ColeMurray/medium-facenet-tutorial#17]
Hi, i a trying to follow this tutorial and it seems that there is an issue with tensorflow

I have successfully generated the intermediate data for the lfw data
I encountered a first issue with downloading the weights (worked around by downloading them aside)

and then an error for the training part.

Could it be an issue with tensorflow being to recent in the docker image ?

INFO:tensorflow:Error reported to Coordinator: <class 'tensorflow.python.framework.errors_impl.InvalidArgumentError'>, assertion failed: [Need value.shape >= size, got ] [18 18 3] [160 160 3]
	 [[Node: random_crop/Assert/Assert = Assert[T=[DT_STRING, DT_INT32, DT_INT32], summarize=3, _device="/job:localhost/replica:0/task:0/cpu:0"](random_crop/All, random_crop/Assert/Assert/data_0, random_crop/Shape, random_crop/size)]]

**Comment by timombre** [comment]
I added a new image in the dataset and now it works.
Maybe a but coming from the use of the dataset the network is already trained on ?

**Comment by ColeMurray** [comment]
Hey @timombre,

It sounds like you resolved the issue. Could you summarize what the problem was and how you resolved it? It's helpful if others encounter a similar issue.

**Comment by timombre** [comment]
Hi @ColeMurray  , 
Thank you for taking time to follow up on your project, it is greatly appreciated :)
I don't really remember what was the issue but I think I was trying to train on an empty dataset or something stupid like that.
Have a good day !
